src/lib/rest/StringFilter.cpp

```diff
@@ -633,13 +609,72 @@ bool StringFilterItem::parse(char* qItem, std::string* errorStringP, StringFilte
 
 /* ****************************************************************************
 *
+* lhsFix - change dots for equals, then remove all quotes
+*/
+static char* lhsFix(char* s)
+{
+  char* scopyP        = strdup(s);
+  char* dotP          = scopyP;
+  bool  insideQuotes  = false;
+  
+  //
+  // Replace ',' for '=' if inside quotes
+  //
+  while (*dotP != 0)
```

Comment:
It's not the same.
This function only changes dots for '=' if inside quotes. Dots outside quoites stay dots.

NTC